src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.SolutionChecksumUpdater.cs

```diff
@@ -146,27 +160,22 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs e)
                 }
             }
 
-            private void CreateInitialSolutionChecksum()
+            private async Task SynchronizeAssets()
             {
-                // initial solution checksum creation won't be affected by global operation.
-                // cancellation can only happen if it is being shutdown.
-                Task.Run(async () =>
-                {
-                    await UpdateSolutionChecksumAsync(ShutdownCancellationToken).ConfigureAwait(false);
+                _synchronize = false;
 
-                    var remoteHostClient = await _service.GetRemoteHostClientAsync(ShutdownCancellationToken).ConfigureAwait(false);
-                    if (remoteHostClient == null)
-                    {
-                        return;
-                    }
+                var remoteHostClient = await _service.GetRemoteHostClientAsync(ShutdownCancellationToken).ConfigureAwait(false);
+                if (remoteHostClient == null)
```

Comment:
I have not been involved in any talks about that.  Why are we thinking about remote-host being disabled in the middle of a VS session?  That will have a significant impact on the IDE and any work we do as it means we must support in-proc and out-of-proc implementations for all our features that we want getting out of the VS address space.

This effectively doubles our testing matrix and means we may have very subtle and hard to track down issues.